src/Pscid/Options.purs

```diff
@@ -72,16 +73,50 @@ mkDefaultOptions =
       <*> mkCommand "test"
       <*> scanDefaultDirectories
 
-mkCommand ∷ ∀ e. String → Eff (fs ∷ FS | e) String
+type IncludePath = String
+
+data CLICommand
+  = NamedScriptSpecificCommand String
+  | NamedScriptCommand String
+  | PulpCommand String (Array IncludePath)
+
+instance showCLICommand :: Show CLICommand where
```

Comment:
Generally I only use `Show` instances for debugging. For a printer like this I prefer a simple function. How about `printCLICommand`?

Reply:
Done.